docs/design/datacontracts/RuntimeTypeSystem.md

```diff
@@ -4,17 +4,64 @@ This contract is for exploring the properties of the runtime types of values on
 
 ## APIs of contract
 
-A `MethodTable` is the runtime representation of the type information about a value.  Given a `TargetPointer` address, the `RuntimeTypeSystem` contract provides a `MethodTableHandle` for querying the `MethodTable`.
+A `MethodTable` is the runtime representation of the type information about a value which is represented as a TypeHandle.
+Given a `TargetPointer` address, the `RuntimeTypeSystem` contract provides a `MethodTableHandle` for querying the `MethodTable`.
+Some other apis are available while using `TypeHandle` which is provided similarly to a MethodTable
+
 
 ``` csharp
 struct MethodTableHandle
 {
     // no public properties or constructors
 
-    internal TargetPointer Address { get; }
+    public TargetPointer Address { get; }
+}
+
+struct TypeHandle
+{
+}
+
+
+internal enum CorElementType
```

Comment:
Instead of spelling this out here, please define the the type and then comment where the values come from.

```csharp
internal enum CorElementType
{
   // Values defined in ECMA-335 - II.23.1.16 Element types used in signatures
}
```